lib/registry.js

```diff
@@ -201,12 +201,24 @@ registry.configureModel = function(ModelCtor, config) {
       'must be an array of objects', modelName);
   }
 
+  // Remote methods
+  if (typeof settings.methods === 'object' && settings.methods !== null) {
+    var methods = settings.methods = settings.methods || {};
+    Object.keys(settings.methods).forEach(function(key) {
+      ModelCtor.remoteMethod(key, settings.methods[key]);
```

Comment:
@bajtos please consider the workspace implementation. Lets not make it impractical to implement the generator via the workspace. I'm not against `prototype.methodName` itself... but we might be making more work for ourselves in the workspace without much benefit.

I'd really like to see remote method creation support added to workspace alongside this addition... These JSON files are specifically designed to serve the workspace well so this new feature is no exception.